client/clip_client/client.py

```diff
@@ -308,7 +309,12 @@ def _prepare_streaming(self, disable, total):
         if total is None:
             total = 500
             warnings.warn(
-                'the length of the input is unknown, the progressbar would not be accurate.'
+                'The length of the input is unknown, the progressbar would not be accurate.'
+            )
+        elif total > 500:
+            warnings.warn(
+                'Input size is too large, it is recommended to use smaller input size for better performance. '
```

Comment:
reject, small input size is not for better performance.

Reply:
iknow, im finding a way of saying this without letting users have a wrong feeling that the service is unstable